README.md

```diff
@@ -51,42 +51,97 @@ print(space_eval(space, best))
 
 ## Contributing 
 
-### Setup
-If you're a developer, clone this repository and install from source:
+### Setup (based on [this](https://scikit-learn.org/stable/developers/contributing.html#contributing-code))
+If you're a developer and wish to contribute, please follow these steps:
+1. Create an account on GitHub if you do not already have one.
+2. Fork the project repository: click on the ‘Fork’ button near the top
+   of the page. This creates a copy of the code under your account on
+   the GitHub user account. For more details on how to fork a repository
+   see [this guide](https://help.github.com/articles/fork-a-repo/).
+3. Clone your fork of the hyperopt repo from your GitHub account to your
+   local disk:
 
 ```bash
-$ git clone https://github.com/hyperopt/hyperopt.git
+$ git clone https://github.com/<github username>/hyperopt.git
 $ cd hyperopt
+```
+
+### 4. Create a virtual env (python 3.x) to contain the dependencies and activate it
+on Windows using virtualenv:  
+`$ python -m venv venv`  
+or with conda  
+`$ conda create --name hyperopt`  
```

Comment:
`conda create -n my_env python=3`

Reply:
and it'd be great to change `venv` on the rest to `my_env` to make it more clear